### DISCUSSION TITLE
Set specific tfvars file

### DISCUSSION THREAD

**Discussion by ramazulay** [terrakube-io/terrakube#789] [Q&A] [answered]
Hi,
How and where should I set my tfvars file(exist in git repo)?

**Comment by alfespa17** [comment]
You could use a file called variables.auto.tfvars so it will automatically detected by terraform

**Reply by ramazulay** [reply]
Thanks for your quick response.
Ok, but if I have three tfvars files per environment, what should I do?
Thanks.

**Reply by alfespa17** [reply]
You could create three different workspaces, each for every environment and use the workspace variables in the UI to set the correct values

**Reply by alfespa17** [reply]
Another option is just having different folders each one with a variables.auto.tfvars

**Reply by ramazulay** [reply]
Hi,
I think i am not clear, lets take a scenario that we have three tfvars files in the same folder(same branch), Is it possible to choose one of them on each environment?
My tfvars file called terraform-QA.tfvars, terraform-DEV.tfvars ..
Can I set env variable that direct to specific tfvars?
For example:
![image](https://github.com/AzBuilder/terrakube/assets/41533452/140714f2-8294-4ac4-bd5d-fa1a97d3bff0)

**Reply by alfespa17** [reply]
There is no way to select one specific tfvars file, but you can set all the variables manually in the UI

**Reply by ramazulay** [reply]
I understand, this feature could be soon?

**Reply by alfespa17** [reply, marked as answer]
You can create the issue description in the repository and we could work on that when we have some free time or if you want con contribute you could also send the pull request with the changes
